tests/integration/standard/test_client_warnings.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2013-2015 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+from cassandra.query import BatchStatement
+from cassandra.cluster import Cluster
+
+from tests.integration import use_singledc, PROTOCOL_VERSION
+
+
+def setup_module():
+    use_singledc()
+
+
+class ClientWarningTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        if PROTOCOL_VERSION < 4:
+            raise unittest.SkipTest(
+                "Native protocol 4,0+ is required for client warnings, currently using %r"
+                % (PROTOCOL_VERSION,))
```

Comment:
Following up, Kishan noticed that this errors in environments utilizing unittest2 instead of unittest. It's [reported fixed in 0.4.2](https://pypi.python.org/pypi/unittest2#id9), but still erroring in 1.0.1